<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoolingGUI</class>
 <widget class="QMainWindow" name="CoolingGUI">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>381</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>791</width>
    <height>381</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cooling control panel</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../udyni.png</normaloff>../udyni.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>751</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cooling control panel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="bn_exit">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>340</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>381</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>17</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string> Evo 1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>57</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Evo 2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>97</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string> Cryo</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>137</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Dryvac</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>175</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TP Spectrometer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>217</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TP VMI</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QAttribute" name="status_evo1" native="true">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QAttribute" name="status_evo2" native="true">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QAttribute" name="status_cryo" native="true">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QAttribute" name="status_dryvac" native="true">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QAttribute" name="status_tp_xuv" native="true">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QAttribute" name="status_tp_vmi" native="true">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="bn_water_xuv">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>170</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bn_water_vmi">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>210</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>361</width>
      <height>251</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Temperature [Â°C]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QAttribute" name="chiller_temp" native="true">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QAttribute" name="chiller_3way" native="true">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>3-way valve [%]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>95</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Fan [%]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QAttribute" name="chiller_fan" native="true">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>135</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Motor [A]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QAttribute" name="chiller_motor" native="true">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QAttribute" name="chiller_state" native="true">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>175</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>State</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="bn_chiller_recovery">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Start recovery</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Chiller Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Valve Status</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAttribute</class>
   <extends>QWidget</extends>
   <header>PyQTango</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
